src/utils/TodoFilter.tsx

```diff
@@ -0,0 +1,23 @@
+import { Todo } from '../types/Todo';
+
+export enum Status {
+  all = 'All',
+  active = 'active',
+  completed = 'completed',
+}
+
+export const Filter = (todos: Todo[], selectedFilter: Status): Todo[] => {
```

Comment:
function name should start with little f
```suggestion
export const getPreparedTodos = (todos: Todo[], selectedFilter: Status): Todo[] => {
```